ray-operator/config/rbac/role.yaml

```diff
@@ -3,6 +3,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  creationTimestamp: null
```

Comment:
Why is this needed?

Reply:
This is auto generated. Not sure the reason

Reply:
This doesn't do anything, so I guess it's fine.